sheriff.go

```diff
@@ -173,6 +173,9 @@ func marshalValue(options *Options, v reflect.Value) (interface{}, error) {
 		k = v.Kind()
 	}
 
+	if k == reflect.Interface {
+		return Marshal(options, val)
+	}
```

Comment:
could you merge that if in an or statement with the next if clause please?

Reply:
done.

Reply:
great! thanks